### PR TITLE
support import of external BA authenticated WmsGeoResources via query parameter

### PR DESCRIPTION
**Pull request by taulinger**

### CODE REVIEW

File: src/services/provider/geoResource.provider.js [hunk 1]
```diff
@@ -158,7 +158,7 @@ export const loadBvvGeoResourceById = id => {
 };
 
 /**
- * Loader for URL-based ID: An URL-based ID must basically match the following pattern:
+ * Loader for external URL-based ID: An URL-based ID must basically match the following pattern:
  * `{url}||{extraParam1}||extraParam1.
```
Comment (thiloSchlemmer) [inline src/services/provider/geoResource.provider.js:162]: did you mean `...{url}||{extraParam1}||extraParam2.`?

**Review by thiloSchlemmer — DISMISSED** [review]
Nice

File: src/services/provider/geoResource.provider.js (hunk 1 above)
Comment (taulinger) [reply src/services/provider/geoResource.provider.js]: Fixed, thx

**Review by thiloSchlemmer — APPROVED** [review]
thx

**Pull request merged**